Tasks/ANTPreview/task.json

```diff
@@ -0,0 +1,168 @@
+        },
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded":false
+        }
+    ],
+    "instanceNameFormat": "AntCC $(options) $(antBuildFile)",
+    "inputs": [
+        { 
+            "name": "antBuildFile", 
+            "type": "filePath", 
+            "label": "Ant build file", 
+            "defaultValue": "", 
+            "required": true,
+            "helpMarkDown": "Relative path from the repository root to the `Ant build file`."
+        },
+        { 
+            "name": "options", 
+            "type": "string", 
+            "label": "Options", 
+            "defaultValue":"", 
+            "required":false 
+        },
+        { 
+            "name": "targets", 
+            "type": "string", 
+            "label": "Target(s)", 
+            "defaultValue":"", "required":false
+        }, 
+        {
+            "name":"publishJUnitResults",
+            "type":"boolean",
+            "label":"Publish to VSO/TFS",
+            "required":true,
+            "defaultValue":"true",
+            "groupName":"junitTestResults",
+            "helpMarkDown":"Select this option to publish JUnit Test results produced by the Ant build to VSO/TFS. Each test result file matching `Test Results Files` will be published as a test run in VSO/TFS."
+        }, 
+        {
+            "name": "testResultsFiles",
+            "type": "filePath",
+            "label": "Test Results Files",
+            "defaultValue": "**/TEST-*.xml",
+            "required": true,
+            "groupName":"junitTestResults",
+            "helpMarkDown": "Test results files path.  Wildcards can be used.  For example, `**/TEST-*.xml` for all xml files whose name starts with TEST-."
+        },
+        {
+            "name": "publishCodeCoverageResults",
+            "type": "boolean",
+            "label": "Publish to VSO/TFS",
+            "required": true,
+            "defaultValue": "false",
+            "groupName": "codeCoverage",
+            "helpMarkDown": "Select this option to publish Code Coverage results produced by the Maven build to VSO/TFS.Code Coverage Summary and code Coverage Files would be published to TFS Server"
+        },
+        {
+            "name": "codeCoverageTool",
+            "type": "pickList",
+            "label": "CodeCoverage Tool",
+            "required": false,
+            "groupName": "codeCoverage",
+            "defaultValue": "JaCoCo",
+            "helpMarkDown": "Select the tool with which summary files have been generated. Only Jacoco support available currently. Jacocoant.jar should be present in lib folder of Ant installation.",
+            "options": {
+                "JaCoCo": "JaCoCo"
+            }
+        },
+        {
+            "name": "summaryFileLocation",
+            "type": "string",
+            "label": "Summary File",
+            "defaultValue": "$(Build.Repository.LocalPath)\\target\\site\\jacoco\\jacoco.xml",
```

Comment:
there can be more than 1 code coverage configured inside ant build file ... how are we planning to address that ?